WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsView.swift

```diff
@@ -32,8 +45,131 @@ struct WooShippingCreateLabelsView: View {
+                            .accessibilityLabel(Localization.BottomSheet.paperSize)
+
+                            Button {
+                                // TODO: 13556 - Trigger purchase action
+                            } label: {
+                                Text(Localization.BottomSheet.purchase)
+                            }
+                            .buttonStyle(PrimaryButtonStyle())
+                            .disabled(true) // TODO: 14044 - Enable button when shipping label is ready to purchase
+                        }
+                        .padding(.horizontal, Layout.bottomSheetPadding)
+                    } else {
+                        Text(Localization.BottomSheet.shipmentDetails)
+                            .foregroundStyle(Color(.primary))
+                            .bold()
+                    }
+                } expandableContent: {
+                    VStack(alignment: .leading, spacing: Layout.bottomSheetSpacing) {
+                        Text(Localization.BottomSheet.orderDetails)
+                            .footnoteStyle()
+
+                        Grid(alignment: .leading, verticalSpacing: .zero) {
+                            GridRow(alignment: .top) {
+                                Text(Localization.BottomSheet.shipFrom)
+                                Text("417 MONTGOMERY ST, SAN FRANCISCO") // TODO: 14044 - Show real "ship from" address (store address)
+                                    .lineLimit(1)
+                                    .truncationMode(.tail)
+                            }
+                            .padding()
+                            Divider()
+                            GridRow(alignment: .top) {
+                                Text(Localization.BottomSheet.shipTo)
+                                VStack(alignment: .leading) {
+                                    Text("1 Infinite Loop") // TODO: 14044 - Show real "ship to" address (customer address)
+                                        .bold()
+                                    Text("Cupertino, CA 95014")
+                                    Text("USA")
+                                }
+                            }
+                            .padding()
+                        }
+                        .font(.subheadline)
+                        .roundedBorder(cornerRadius: Layout.cornerRadius, lineColor: Color(.separator), lineWidth: 0.5)
+                        Group {
+                            AdaptiveStack {
+                                Image(uiImage: .productIcon)
+                                    .frame(width: Layout.iconSize)
+                                Text(viewModel.items.itemsCountLabel)
+                                    .bold()
+                                Spacer()
+                                Text("$148.50") // TODO: 14044 - Show real item total
```

Comment:
We already calculate this in `viewModel.items` so we might want to pull the data from there, since we expect the label to be the same in both places on the screen.